src/Sulu/Component/Webspace/Tests/Unit/Manager/WebspaceCollectionCache.php

```diff
@@ -0,0 +1,1118 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Webspace\Tests\Unit\Manager;
+
+use Sulu\Component\Localization\Localization;
+use Sulu\Component\Webspace\Environment;
+use Sulu\Component\Webspace\Manager\WebspaceCollection;
+use Sulu\Component\Webspace\Navigation;
+use Sulu\Component\Webspace\NavigationContext;
+use Sulu\Component\Webspace\Portal;
+use Sulu\Component\Webspace\PortalInformation;
+use Sulu\Component\Webspace\Security;
+use Sulu\Component\Webspace\Segment;
+use Sulu\Component\Webspace\Url;
+use Sulu\Component\Webspace\Webspace;
+
+/**
+ * WebspaceCollectionCache65db58eff00cf.
+ *
+ * DO NOT EDIT
+ * This file is autogenerated by the Sulu Webspace component
+ */
```

Comment:
This is a cache file from the last version to check that the new processes are compatible with the old behaviour.